governance/delegates/0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv/profile.md

```diff
@@ -0,0 +1,9 @@
+---
+name: WBC
+external_profile_url: https://forum.makerdao.com/t/wbc-aligned-delegate-communications/20828
+---
+
+# WBC
+Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2bvv
```

Comment:
The delegate's address has two extra 'v'. Additionally, the name of the folder containing it has the same address with the same error. I have reviewed the transactions, and this delegate created this contract: 0xc139042755eae94bd10798251801b5bd26464b69. 
I believe the folder should have that name. Could you also check it to be sure, as the error is repeated in the MIPs? Thank you.
`Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2b`

Reply:
```suggestion
Delegate Address: 0x3C85C3F4B0d610Fd29cc132A2073f85C15F83f2b
```

Reply:
https://github.com/makerdao/mips/blob/master/MIP113/MIP113.md?plain=1#L374

It is correct in the MIP (https://github.com/makerdao/mips/blob/master/MIP113/MIP113.md?plain=1#L374) we think this is just a copy-paste artefact. Good spot.